docs/reference/announcements.md

```diff
@@ -10,6 +10,16 @@ Scheduled release date: 8th of Oct 2024
 
 Scheduled end of maintenance: 14th of April 2026
 
+### Configuring license keys
+
+Camunda 8 components are now able to consume Enterprise licensing information. For additional details, review the [blog post on licensing updates for Camunda 8 Self-Managed](https://camunda.com/blog/2024/04/licensing-update-camunda-8-self-managed/).
+
+Review the documentation for your components for more information on how to provide the license key to each component as an environment variable. To configure with Helm, visit the [Self Managed installation documentation](/self-managed/setup/install.md).
```

Comment:
Would it make sense to link all the sub-pages from here for easier discovery?

Reply:
Added in latest commit 👍